### DISCUSSION TITLE
Inquiry on Using Gotenberg-Webhook-Url Header for Async SVG to PDF Conversion

### DISCUSSION THREAD

**Discussion by VISHNUDAS-tunerlabs** [gotenberg/gotenberg#935] [Q&A]
I am attempting to convert SVG into PDF format using the specified endpoint. We are able to do it in sync mode. I want to call this endpoint in async mode. Could you please confirm if the Gotenberg-Webhook-Url header, as mentioned in the documentation [here](https://gotenberg.dev/docs/webhook), works for the forms/libreoffice/convert endpoint?

Thank you for your assistance.

**Comment by gulien** [comment]
Hello @VISHNUDAS-tunerlabs,

Indeed, it works for all `multipart/form-data` 👍

**Reply by VISHNUDAS-tunerlabs** [reply]
@gulien so in the request header we have to specify content-type as multipart/form-data ?

**Reply by gulien** [reply]
I mean if it works in a synchronous fashion, just add the webhook header(s) and that’s it.

**Reply by VISHNUDAS-tunerlabs** [reply]
@gulien Thank you. One more clarification. once the callback happens where we can find

`Content-Disposition: attachment; filename={output-filename.ext}
Content-Type: {content-type}
Content-Length: {content-length}
Gotenberg-Trace: {trace}
User-Agent: Gotenberg
Body: {output-file}`

there data? req.body? currently I am getting req.body as {}

**Reply by gulien** [reply]
If it’s successful, it should be in the body 🤔
